tests/integration/grpc_plugin/BUILD.bazel

```diff
@@ -0,0 +1,39 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+load("@io_bazel_rules_go//proto:def.bzl", "go_proto_library")
+load("@rules_proto//proto:defs.bzl", "proto_library")
+
+proto_library(
+    name = "hello_proto",
+    srcs = ["hello.proto"],
+    deps = [],
+)
+
+go_proto_library(
```

Comment:
I considered putting this in a subdirectory so that we can add other tests specifically around the grpc_plugin without conflicts. I opted not to — it's an easy enough refactor if the need arises in the future